feeds/backend.php

```diff
@@ -30,21 +33,45 @@
         switch($content) {
             case "posted":
                 $condition = sprintf("state='%s'", PROJ_SUBMIT_PG_POSTED);
```

Comment:
We could go back to `$state` instead of `$condition` since the special case of SR is being handled in a different way. Not sure if we _want_ to though.

Reply:
Nah, I don't see a need to change it back.